initramfs/init_functions.sh

```diff
@@ -49,18 +49,34 @@ setup_usb_configfs() {
 	echo "rndis" > $CONFIGFS/g1/configs/c.1/strings/0x409/configuration \
 		|| echo "  Couldn't write configration name"
 
-	# Make sure the node for the eMMC exists
-	if [ -z "$(ls $EMMC)" ]; then
-		fatal_error "$EMMC could not be opened, possible eMMC defect"
+	if [[ $EMMC != "" ]]; then
```

Comment:
This should be
```bash
if [ -n "$EMMC" ]; then
...
```

Reply:
And likewise for the other similar IF blocks

Reply:
it was first, but it didn't work :woman_shrugging:

Reply:
oh really? oops,,, maybe `[[ -n "$EMMC" ]]` ?

Reply:
both don't work. i'll just leave it like it is because that works